src/table/components/TableHeadWrapper.tsx

```diff
@@ -88,6 +79,32 @@ function TableHeadWrapper({
                   </VisuallyHidden>
                 )}
               </StyledSortLabel>
+            );
+          };
+
+          return (
+            <StyledHeadCell
+              sx={{ ...headerStyle, zIndex: columns.length - columnIndex }}
+              key={column.id}
+              align={column.align}
+              className="sn-table-head-cell sn-table-cell"
+              tabIndex={tabIndex}
+              aria-sort={ariaSort}
+              aria-pressed={isCurrentColumnActive}
+              onKeyDown={handleKeyDown}
+              onMouseDown={() => handleClickToFocusHead(columnIndex, rootElement, setFocusedCellCoord, keyboard)}
+              onClick={(evt: React.MouseEvent) => handleClickToSort(evt, column, changeSortOrder, isInteractionEnabled)}
+            >
+              {areBasicFeaturesEnabled ? (
+                <HeadCellContent>
+                  <HeadCell />
+                  <HeadCellMenu headerStyle={headerStyle} translator={translator} />
+                </HeadCellContent>
+              ) : (
+                <HeadCellContent>
+                  <HeadCell />
+                </HeadCellContent>
+              )}
```

Comment:
```suggestion
              <HeadCellContent>
                <StyledSortLabel
                  headerStyle={headerStyle}
                  active={isCurrentColumnActive}
                  title={!constraints.passive ? column.sortDirection : undefined} // passive: turn off tooltips.
                  direction={column.sortDirection}
                  tabIndex={-1}
                  onMouseDown={(evt: React.MouseEvent) =>
                    handleMouseDownLabelToFocusHeadCell(evt, rootElement, columnIndex)
                  }
                >
                  <CellText>{column.label}</CellText>
                  {isFocusInHead && (
                    <VisuallyHidden data-testid={`VHL-for-col-${columnIndex}`}>
                      {translator.get('SNTable.SortLabel.PressSpaceToSort')}
                    </VisuallyHidden>
                  )}
                </StyledSortLabel>
                {areBasicFeaturesEnabled && <HeadCellMenu headerStyle={headerStyle} translator={translator} />}
              </HeadCellContent>
```

Reply:
AAh, yes off course! I forgot to refactor it when both are with HeadCellContent